src/modules/dashboard/components/ColonyHomeActions/ColonyActionsItem.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import Icon from '~core/Icon';
+import Paragraph from '~core/Paragraph';
+import styles from './ColonyActionsItem.css';
+
+interface Props {
+  title: object;
+  description: object;
+  disabled?: boolean;
+}
+
+const MSG = defineMessages({
+  coming: {
+    id: 'dashboard.ColonyHomeActions.ColonyActionsItem.coming',
+    defaultMessage: 'Coming',
+  },
+});
+
+const ColonyActionsItem = ({ title, description, disabled }: Props) => {
```

Comment:
This should be named `ColonyActionsDialogItem` as it is a child of `ColonyActionsDialog`, not `ColonyActions`